plugins/com.redhat.fabric8analytics.lsp.eclipse.ui/src/com/redhat/fabric8analytics/lsp/eclipse/ui/Fabric8AnalyticsStreamConnectionProvider.java

```diff
@@ -161,7 +161,7 @@ private static File getNodeJsLocation() {
 	private static File getServerLocation() {
 		try {
 			Bundle bundle = Platform.getBundle(Fabric8AnalysisLSUIActivator.PLUGIN_ID);
-			return new File(FileLocator.getBundleFile(bundle), "/server/fabric8-analytics-lsp-server-master/output/server.js");
+			return new File(FileLocator.getBundleFile(bundle), "ca-lsp-server/server.js");
```

Comment:
Is the whole point here to fetch a single .js file? If so, why are we bothering to tar up the server if we just need a single file?

Reply:
Because we are running the server from the Eclipse as a separate process.